docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,162 @@
+# Publish GOV.UK Frontend when `main` branch has unreleasable changes
+
+Follow this document to publish a support branch of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix as part of a:
```

Comment:
```suggestion
This document is for GOV.UK Design System developers who need to publish a support branch of GOV.UK Frontend when the `main` branch contains unreleasable changes. For example, you might need to release a fix as part of a:
```
@vanitabarrett We thought it would be good to call out who the doc is for. Can it be for any other roles besides Design System devs?

Reply:
@EoinShaughnessy I don't think so - only developers have access to the NPM credentials via the design system credentials repo at the moment.

Reply:
@vanitabarrett Thanks!

@jonathanglassman How's the suggested wording?

Reply:
@EoinShaughnessy works for me